test_scripts/Capabilities/PersistingHMICapabilities/032_Suspending_of_RAI_request_processing_during_OnReady_with_another_ccpu_version_in_case_cash_file_exists.lua

```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0249-Persisting-HMI-Capabilities-specific-to-headunit.md
+--
+-- Check that SDL suspend of RAI request processing from mobile app until all HMI Capabilities
+--  (VR/TTS/RC/UI/Buttons.GetCapabilities/,VR/TTS/UI.GetSupportedLanguages/GetLanguage, VehicleInfo.GetVehicleType)
+--  are received from HMI in case mobile device connected during OnReady communication,
+--  HMI capabilities cache file (hmi_capabilities_cache.json) exists on file system and ccpu_version do not match
+--
+-- Preconditions:
+-- 1  Value of HMICapabilitiesCacheFile parameter is defined (hmi_capabilities_cache.json) in smartDeviceLink.ini file
+-- 2. HMI sends GetSystemInfo with ccpu_version = "ccpu_version_1" to SDL
+-- 3. HMI sends all capabilities to SDL
+-- 4. SDL persists capabilities to HMI capabilities cache file ("hmi_capabilities_cache.json") in AppStorageFolder
+-- 5. Ignition OFF/ON cycle performed
+-- 6. SDL is started and send GetSystemInfo request
+-- Sequence:
+-- 1. Mobile connected during during OnReady communication and sends RegisterAppInterface request to SDL
```

Comment:
@aderiabin `OnReady` is a notification, phrase `OnReady communication` is a bit confusing.

Reply:
@AByzhynar Rephrased